PurchasesTests/Mocks/SKProductSubscriptionDurationExtensions.swift

```diff
@@ -8,11 +8,12 @@
 
 import Foundation
 import StoreKit
```

Comment:
We don't need to import StoreKit anymore I think

Reply:
I thought we did? I'll check

Reply:
Not a biggie, it's just because it doesn't look like we directly reference anything in StoreKit

Reply:
Actually we do `SKProductSubscriptionPeriod` haha